### PR TITLE
Add Missing Help Entries

### PR DESCRIPTION
**Pull request by kingthorin**
Update pscanrules.html adding help content for numerous release quality
passive scanners that were missing descriptions.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html [hunk 1]
```diff
@@ -11,24 +11,40 @@
...
  
 <H2>Cache Control</H2>
+Checks "Cache-Control" and "Pragma" response headers against general industry best practice settings for protection of sensitive content. 
 
 <H2>Content Type Missing</H2>
+Raises an alert if the response is lacking a Content-Type header or if the header exists but the value is empty.
 
 <H2>Cookie HTTP Only</H2>
+Ensures that as cookies are set they are flagged HttpOnly. The HttpOnly flag indicates to browsers that the cookie 
+being set should be be acted upon by client side script (such as JavaScript).
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html]: The word "be" is repeated, typo?

File: src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html [hunk 2]
```diff
@@ -11,24 +11,40 @@
...
 <H2>Cross Domain Script Inclusion</H2>
+Validates whether or not scripts are included from domains other than the domain hosting the content. 
+By looking at the "src" attributes of "script" tags in the response.
 
 <H2>Header XSS Protection</H2>
+Checks for the existence of and value/setting of the X-XSS-Protection header. 
+This response header can be used to configure a user-agent's built in reflective XSS protection.
 
 <H2>Mixed Content</H2>
+For content served via HTTPS analyze all the src attributes in the response looking for those sourced
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html]: "analyse" instead of "analyze"? ;)

**Comment by kingthorin** [comment]
Fixed

File: src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html [hunk 3]
```diff
@@ -11,24 +11,40 @@
...
+ Check server responses for HTTP 500 - Internal Server Error type responses or those that contain a known error string.
  
 <H2>Cache Control</H2>
+Checks "Cache-Control" and "Pragma" response headers against general industry best practice settings for protection of sensitive content. 
 
 <H2>Content Type Missing</H2>
+Raises an alert if the response is lacking a Content-Type header or if the header exists but the value is empty.
 
-<H2>Cookie HTTP Only</H2>
+<H2>Cookie HTTPOnly</H2>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html]: Was this change on purpose? If so, "HttpOnly" instead? Or, the actual scanner name "Cookie No HttpOnly Flag"?

Comment (kingthorin) [inline src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html]: Changed it to be consistent with content. Fixup done.

Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html]: Yeah, makes sense.

**Comment by thc202** [comment]
Looks good to me.

**Comment by psiinon** [comment]
Nice :)
